libs/server/Custom/CustomCommandManagerSession.cs

```diff
@@ -12,25 +12,40 @@ internal sealed class CustomCommandManagerSession
     {
         readonly CustomCommandManager customCommandManager;
         public readonly (CustomTransactionProcedure, int)[] sessionTransactionProcMap;
+        public readonly CustomProcedure[] sessionCustomProcMap;
```

Comment:
These 2 arrays can be made private, right? Also, might be good to add a comment that the arrays are indexed by the IDs.

Reply:
Fixed.